src/github/deployment.ts

```diff
@@ -39,11 +38,6 @@ export const processDeployment = async (
 		webhookReceived: webhookReceivedDate
 	});
 
-	if (await isBlocked(gitHubInstallationId, logger)) {
```

Comment:
No need to put it here as the webhook middleware will block at the WebServer side before entering into queue.